valqueries-automapper/src/main/java/com/valqueries/automapper/schema/ValqueriesSchemaExecutor.java

```diff
@@ -39,6 +40,11 @@ public void execute(Collection<TableAction> collection) {
 		}
 	}
 
+	@Override
+	public void execute(Collection<TableAction> collection, DataSource dataSource) {
+		throw new UnsupportedOperationException("aaaaaaaaaa");
```

Comment:
@emanuel-persequor you approved this, right?

Reply:
haha, I couldn't find anywhere it was invoked - so ended up forgetting about it when reading through the changed files.